packages/cloud-cognitive/src/components/Toolbar/_toolbar.scss

```
@@ -8,7 +8,7 @@
 @import 'carbon-components/scss/components/button/button';
 @import '../../global/styles/project-settings';
 
-@include exports($name: cloud-cognitive--toolbar) {
+@include exports('toolbar') {
```

These styles will never be exported due to [naming conflicts with Carbon's toolbar](https://github.com/carbon-design-system/ibm-cloud-cognitive/blob/main/packages/cloud-cognitive/src/global/styles/_imported-carbon-modules.scss#L23).

```suggestion
@include exports('ibm-products--toolbar') {
```

---

Thanks, good catch